ui/partials/perf/compareconfidence.html

```diff
@@ -1,3 +1,8 @@
-<span ng-if="text && value">
-  {{value|displayPrecision}} (<abbr title="Result of running t-test on base versus new result distribution: higher values indicate greater confidence that the indicated performance difference is statistically meaningful">{{text}}</abbr>)
+<span ng-if="result.confidenceText && result.confidence">
+  {{result.confidence|displayPrecision}} (<abbr title="Result of running t-test on base versus new result distribution: higher values indicate greater confidence that the indicated performance difference is statistically meaningful">{{result.confidenceText}}</abbr>)
+  <span ng-if="result.originalRuns.length < 5 || result.newRuns.length < 5"
+        class="glyphicon glyphicon-warning-sign text-warning"
+        tooltip="Need at least 5 runs for each push (base and new) to have full confidence in the comparison"
```

Comment:
Hmm.. there's never "full confidence", and low number of runs just reduces the confidence level (which manifests at the t-test column value).

How about something like "Low confidence level due to too few runs. Please consider retriggering the test to have at least 5 runs (for both base and new)" ?